packages/zowe-explorer-api/src/tree/ZoweTreeNode.ts

```diff
@@ -74,8 +75,11 @@ export class ZoweTreeNode extends vscode.TreeItem {
      *
      * @returns {imperative.IProfileLoaded}
      */
-    public getProfile(): imperative.IProfileLoaded {
-        return this.profile ?? this.getParent()?.getProfile();
+    public getProfile(profilesCache?: ProfilesCache): imperative.IProfileLoaded {
```

Comment:
I noticed that the function definition was updated for this class but the interface is not. Do we want to allow extenders to pass this `profilesCache` to `getProfile` in their own tree node implementations?